src/metabase/api/dashboard.clj

```diff
@@ -846,6 +847,54 @@
 
 ;;;;;;;;;;;;;;;;;;;;; End functions to handle broken subscriptions
 
+(defn create-editable-table-card!
```

Comment:
Wasn't sure about whether to move this into the dashcard model namespace. Let's see.